doc/functions/country_names.rst

```diff
@@ -0,0 +1,38 @@
+``country_names``
+=====================
+
+The ``country_names`` function returns the names of the countries:
+
+.. code-block:: twig
+
+    {# Afghanistan, Åland Islands, ... #}
+    {{ country_names()|join(', ') }}
+    
+By default, the function uses the current locale. You can pass it explicitly:
+
+.. code-block:: twig
+
+    {# Afghanistan, Afrique du Sud, ... #}
+    {{ country_names('fr')|join(', ') }}
+
+.. note::
+
+    The ``country_names`` function is part of the ``IntlExtension`` which is not
+    installed by default. Install it first:
+
+    .. code-block:: bash
+
+        $ composer require twig/intl-extra
+
+    Then, on Symfony projects, install the ``twig/extra-bundle``:
+
+    .. code-block:: bash
+
+        $ composer require twig/extra-bundle
+
+    Otherwise, add the extension explicitly on the Twig environment::
+
+        use Twig\Extra\Intl\IntlExtension;
+
+        $twig = new \Twig\Environment(...);
+        $twig->addExtension(new IntlExtension());
```

Comment:
Can you add an `Arguments` section like for other functions on all doc files? Something like:

```

Arguments
---------

* ``locale``: The locale
```